### DISCUSSION TITLE
Pixel 6 Volte show up but not working on Vinaphone

### DISCUSSION THREAD

**Discussion by Puddiiing** [kyujin-cho/pixel-volte-patch#85] [Carrier-specific Experiences]
Hi, 
I managed to install 1.2.3 and got Vowifi working fine even with airplane mode turn on. However, Volte showed up and checked but not working at all, calls fell back to 3G. Currently on latest February security update.
Pixel 6 
Carrier: Vinaphone-VietNam

**Comment by Puddiiing** [comment]
Solved by downgrading to ver 1.2.2. Both VoLTE and VoWifi work flawlessly. 😍😍
@kyujin-cho Please pay attention to ver 1.2.3. Something might be broken

**Reply by kszaq** [reply]
FWIW 1.2.3 Works for me. Would you be able to provide `adb shell dumpsys carrier_config` both from 1.2.2 (working) and 1.2.3 (not working)?
